java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6355_java.json

```diff
@@ -12,6 +12,5 @@
   "defaultSeverity": "Major",
   "ruleSpecification": "RSPEC-6355",
   "sqKey": "S6355",
-  "scope": "All",
-  "defaultQualityProfiles": ["Sonar way"]
```

Comment:
The rule will still be in the default profile, this line is still useless as `Sonar_way_profile.json` is the reference to know if a rule is in the default profile.